README.md

```diff
@@ -19,17 +19,17 @@ Want to just get started now? Check out the [instructions](#getting-started).
 
 1. Simplify the process of application source + builder image -> usable image for most use cases (the
    80%)
-1. Define and implement a workflow for incremental build that eventually uses only docker
+1. Define and implement a workflow for incremental builds that eventually uses only Docker
```

Comment:
uses is correct... the noun "uses" applies to is "workflow" and not "builds", i believe.  or at least that argument can be made..it's a confusing sentence :)